commands/developer-utils/github-open-pull-requests-details.template.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Show Open Pull Requests
+# @raycast.mode fullOutput
+#
+# Optional parameters:
+# @raycast.packageName GitHub
+# @raycast.icon images/github-logo.png
+
+require 'json'
+require 'net/http'
+require 'uri'
+
+API_TOKEN = ''
+
+if API_TOKEN.empty?
+  puts 'No API token provided'
+  exit(1)
+end
+
+uri = URI('https://api.github.com/graphql')
+req = Net::HTTP::Post.new(uri)
+req['Authorization'] = "token #{API_TOKEN}"
+req.body = '{ "query": "query { viewer { pullRequests(first: 10 states: OPEN) { nodes {baseRepository {name} title number url} } } }" }'
```

Comment:
To keep the consistency suggested by Thomas, lets add some more spaces here.
```suggestion
req.body = '{ "query": "query { viewer { pullRequests(first: 10 states: OPEN) { nodes { baseRepository { name } title number url } } } }" }'
```

Reply:
Thanks for the suggestions, you are right in that it makes it more consistent! I didn't know how Ruby deals with whitespace, but looks like it's fine and I agree that this makes it look a lot better

Reply:
Actually in this case isn't Ruby, but GraphQL. It's nice to have some spaces just to keep the query nice in the Script Command. It won't hurt any rule from Ruby, but as you added some spaces in other queries, looks nice to have them also here! 😄